tests/unit/moduleapi/list.tcl

```diff
@@ -63,4 +63,24 @@ start_server {tags {"modules"}} {
         r list.edit k reverse rkr foo bar
         r list.getall k
     } {bar y foo}
+
+    proc sleep {time} {
+        after $time set end 1
+        vwait end
+    }
+
+    test {Busy module} {
+        set rd [redis_deferring_client]
+        $rd list.busy
+        catch {r ping} e
+        assert_match {BUSY*} $e
+        catch {r ping} e
```

Comment:
maybe we can add some timing, to see that we didn't get here too soon?

Reply:
@oranagra  I'm not sure I understand, we want to see the busy error... 
what do you mean by timing ?

Reply:
something like this, to measure time, and see that it's bigger than the script-time-limit we've set
```
set start [clock clicks -milliseconds]
set elapsed [expr [clock clicks -milliseconds]-$start]
```